### PR TITLE
Icelake client

### PR DESCRIPTION
**Pull request by jeffhammond**
Updating #370

### CODE REVIEW

File: configure [hunk 1]
```diff
@@ -1499,6 +1506,9 @@ check_compiler()
 				#blacklistcc_add "zen"
 				: # explicit no-op since bash can't handle empty loop bodies.
 			fi
+			if [ ${cc_major} -lt 6 ]; then
```
Comment (isuruf) [inline configure:1509]: Clang 6 is Apple clang 10. You'll need to add that condition to Line 1486.

Comment (jeffhammond) [reply configure:1509]: I am not willing to blacklist a compiler I cannot test.  When I have an Apple system with Ice Lake, I'll revisit this.  I am not going to devote any effort to dealing with Apple's breaking of the LLVM toolchain versioning.

**Comment by fgvanzee** [comment]
I did a quick review of your work. It looks like you either magically knew with files to change, or found my [ConfigurationHowTo.md](https://github.com/flame/blis/blob/master/docs/ConfigurationHowTo.md) helpful. :)

**Comment by jeffhammond** [comment]
> I did a quick review of your work. It looks like you either magically knew
> with files to change, or found my ConfigurationHowTo.md
> <https://github.com/flame/blis/blob/master/docs/ConfigurationHowTo.md>
> helpful. :)
>

I can’t remember but I think I just figured it out by brute force pattern
recognition. While I greatly respect your documentation efforts, I
benefitted more from the clean design of the file hierarchy.

This is just a refresh of what I did over the winter holiday. I need to
validate performance and then I’ll release it. I finally got around to
benchmarking against MKL.

> --
Jeff Hammond
jeff.science@gmail.com
http://jeffhammond.github.io/

**Comment by fgvanzee** [comment]
Boo (and thanks). Then I'll use my documentation to make sure you didn't miss anything. :)

**Pull request closed without merging**